samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java

```diff
@@ -0,0 +1,25 @@
+package com.hexascribe.vertexai.controller;
+
+import com.hexascribe.vertexai.service.VertexService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/vertex")
+public class VertexController {
+
+    @Autowired
+    private VertexService vertexService;
+
+    @GetMapping("prediction")
```

Comment:
This mapping requires `/` as a prefix.

Reply:
The @RequestMapping annotation already defines the base URL mapping for the class, so it doesn't need to explicitly specify the "/" as a prefix in the `getMapping`